tt_llk_blackhole/common/inc/sfpu/ckernel_sfpu_where.h

```diff
@@ -0,0 +1,100 @@
+// SPDX-FileCopyrightText: © 2025 Tenstorrent AI ULC
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "sfpi.h"
+
+namespace ckernel
+{
+namespace sfpu
+{
+
+/*
+Expects following input in Dst register:
+Index 0 ( Tile 0 ) -> condition tensor
+Index 32 ( Tile 1 ) -> true tensor
+Index 64 ( Tile 2 ) -> false tensor
+
+*/
+
+template <bool APPROXIMATION_MODE, int ITERATIONS = 32>
+inline void _calculate_where_fp16_b()
```

Comment:
```suggestion
inline void _calculate_where_fp16_b_()
```
Stick to the convention please :)